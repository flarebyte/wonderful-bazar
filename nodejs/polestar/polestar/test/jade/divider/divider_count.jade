include mixins/divider
ul
  mixin divider_count(everything.title, everything.items)
  mixin divider_count(no_items.title, no_items.items)
  mixin divider_count(items_is_null.title, items_is_null.items)
  mixin divider_count(items_is_empty.title, items_is_empty.items)
  
  
