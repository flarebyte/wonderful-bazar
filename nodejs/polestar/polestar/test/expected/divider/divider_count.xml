<ul><li data-role="list-divider" data-theme="f">everything<span class="ui-li-count">3</span></li><li data-role="list-divider" data-theme="f">items_is_empty<span class="ui-li-count">0</span></li></ul>